README.md

```diff
@@ -78,25 +80,45 @@ Then, make each route respond with a one-word string containing the RESTful acti
 var express = require("express");
 var app = express();
 
-// Your code starts here...
+var bodyParser = require("body parser"); //neded for post requests
 
-```
+app.get("/", function(err, res){
+  Model.find({}).then(function(models){
+    console.log("Welcome");
+    });
+  });
+
+app.get("/models/:name", function(err, res){
+  Model.findOne({name: req.params.name}).then(function(model){
+    console.log("Show");
+    });
+  });
+
+app.put("/models/:name", function(err, res){
+  Model.findOneAndUpdate({name: req.params.name}, req.body.model, {new: true}).then(function(model){
+    console.log("Edit");
+    });
+  });
+
+app.delete("/models/:name/delete", function(err, res){
+  Model.findOneAndRemove({name: req.params.name}).then(function(){
+    console.log("Delete");
+    });
+  });
```

Comment:
:+1:
I'm not sure how to grade this one.  You put a lot of effort into representing the mongoose functionality, but we didn't ask for that.  We asked you to "make each route respond with a one-word string containing the RESTful action that would most likely be associated with this route."  These routes don't do that.  They console.log that word instead of responding with the word.  The http methods pass request and response, not (err, response).  The mongoose methods pass error and result.